metricbeat/docs/modules/mysql.asciidoc

```diff
@@ -55,14 +55,25 @@ metricbeat.modules:
   # The username and password can either be set in the DSN or using the username
   # and password config options. Those specified in the DSN take precedence.
   hosts: ["root:secret@tcp(127.0.0.1:3306)/"]
+
+
+- module: mysql
+  metricsets: ["query"]
+  namespace: "test"
+  query: "SELECT * FROM example"
```

Comment:
I see we have a bit of a config issue here, as the `namespace` and `query` apply only to the `query` metricset but structurally the config seems to indicate they apply to the whole module. But that's a more general Metricbeat issue, right?